src/README.md

```diff
@@ -1,38 +1,24 @@
-<!--
-  -- Copyright (c) 2019, Arm Limited, All Rights Reserved
-  -- SPDX-License-Identifier: Apache-2.0
-  --
-  -- Licensed under the Apache License, Version 2.0 (the "License"); you may
-  -- not use this file except in compliance with the License.
-  -- You may obtain a copy of the License at
-  --
-  -- http://www.apache.org/licenses/LICENSE-2.0
-  --
-  -- Unless required by applicable law or agreed to in writing, software
-  -- distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-  -- WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  -- See the License for the specific language governing permissions and
-  -- limitations under the License.
---->
-
 # Parsec
 
-**Parsec** is the **P**latform **A**bst**R**action for **SEC**urity,
-a new open-source initiative to provide a common API to secure services in a platform-agnostic way.
+**Parsec** is the **P**latform **A**bst**R**action for **SEC**urity, a new open-source initiative to
+provide a common API to secure services in a platform-agnostic way.
 
 Find here all the technical documentation of Parsec, alongside user and developer guides.
 
 # Disclaimer
 
 Parsec is a new open source project and is under development. This code repository is being made
-available so that the developer community can learn and give feedback about the new interfaces and the concepts of platform-agnostic security.
-The implementation that is provided is suitable for exploratory testing and experimentation only.
-This test implementation does not offer any tangible security benefits and therefore is not
-suitable for use in production. Documentation pages may be incomplete and are subject to change
-without notice. Interfaces may change in such a way as to break compatibility with client code.
-Contributions from the developer community are welcome. Please refer to the contribution guidelines.
+available so that the developer community can learn and give feedback about the new interfaces and
+the concepts of platform-agnostic security. The implementation that is provided is suitable for
+exploratory testing and experimentation only. This test implementation does not offer any tangible
+security benefits and therefore is not suitable for use in production. Documentation pages may be
+incomplete and are subject to change without notice. Interfaces may change in such a way as to break
+compatibility with client code. Contributions from the developer community are welcome. Please refer
+to the contribution guidelines.
 
 # License
 
-Parsec and all the repositories under the `parallaxsecond` organization are provided under Apache-2.0.
-Contributions to this project are accepted under the same license.
+Parsec and all the repositories under the `parallaxsecond` organization are provided under
+Apache-2.0. Contributions to this project are accepted under the same license.
+
+*Copyright (c) 2019, Arm Limited. All rights reserved.*
```

Comment:
Shouldn't this be 2020 since you now changed the file?

Reply:
We are currently discussing ways of simplifying the copyright notice and will update it to 2019-2020 depending on how the discussion goes.